image-toolkit/sbin/startup.sh

```diff
@@ -53,6 +53,49 @@ vlc_restart_loop() {
     done
 }
 
+webcam_stream_loop() {
+    VIDEO_DEVICE_NO=0
```

Comment:
Due to https://github.com/obsproject/obs-studio/issues/3003 (webcam numbering may differ after reconnection), use devices in `/dev/v4l/by-id`